src/shared/components/structureViewer/StructureViewerPanel.tsx

```diff
@@ -508,6 +530,24 @@ export default class StructureViewerPanel extends React.Component<IStructureView
         }
     }
 
+    @computed get structureViewerBounds(): {width: number|string, height: number|string} {
+        let width: number|string;
+        let height: number|string;
+
+        // if 3Dmol container div is not initialized yet, just set to a default value: width=auto; height=350
+        // otherwise toggle the size
+        if (this.isIncreasedSize) {
```

Comment:
But why does it ignore the observable `isDoubleSize`? Whenever it changes, `computed` should recompute, but it doesn't if you write it this way...